.babelrc

```diff
@@ -9,5 +9,6 @@
       }
     ],
     "@babel/preset-typescript"
-  ]
+  ],
+  "plugins": [["@babel/plugin-proposal-decorators", { "legacy": true }]]
```

Comment:
I just did some noodling and couldn't figure it out. Using `legacy:true` is the suggestion at https://babeljs.io/docs/en/babel-plugin-transform-typescript and I think that's where I got it

![image](https://user-images.githubusercontent.com/2023/117315603-d68dc300-ae55-11eb-8eaf-4b865ae8c9d2.png)

According to the [babel-plugin-proposal-decorators docs](https://babeljs.io/docs/en/babel-plugin-proposal-decorators), `legacy: true` enables the "stage 1" decorators.

![image](https://user-images.githubusercontent.com/2023/117316111-4ac86680-ae56-11eb-8558-4c76ef9d9665.png)

Meanwhile, the official [TS Docs on Decorators](https://www.typescriptlang.org/docs/handbook/decorators.html) mention that decorators are "stage 2", implying that TS supports stage-2 decorator syntax.

When I mess around with this locally I can remove the "legacy: true" and the code appears to work correctly if I try it out (with `ts-node`), and `yarn build` (aka, `tsc`) is happy.
However, when I do that, the `jest` tests don't work correctly, I get syntax or compile errors.
I think there is something I am not understanding about the difference between jest's TS support and our TS compilation.

Anyway, to avoid going further into the rabbithole I'll just remove this decorator altogether. It's only used in a few places.

Reply:
removed!